### ISSUE TITLE
Postgres DB location on disk

### ISSUE THREAD

**Issue by yazeed** [laravel/homestead#1574]
Again, this is just a question...

Upon running, **df -h** I notice there's a designated mysql disk in the list, but it seems Postgres does not save it's databases there, so I was wondering if this was an area that's been missed after Postgres was added, and if Postgres database location on disk should belong there on the mysql disk, or even on it's own.

If so, I wouldn't exactly know how to submit a PR for this one, but maybe someone else can.

I believe having it seperated from the main disk can help in managing space expectations around the vagrant build.

```
Filesystem                               Size  Used Avail Use% Mounted on
udev                                     5.9G     0  5.9G   0% /dev
tmpfs                                    1.2G  1.5M  1.2G   1% /run
**/dev/mapper/homestead--vg-root            62G   52G  7.1G  88% /**
tmpfs                                    5.9G     0  5.9G   0% /dev/shm
tmpfs                                    5.0M     0  5.0M   0% /run/lock
tmpfs                                    5.9G     0  5.9G   0% /sys/fs/cgroup
/dev/sda1                                511M  4.0K  511M   1% /boot/efi
**/dev/mapper/homestead--vg-mysql--master   63G  222M   60G   1% /homestead-vg/master**
/dev/loop2                                56M   56M     0 100% /snap/core18/1932
/dev/loop1                               119M  119M     0 100% /snap/chromium/1424
/dev/loop0                                56M   56M     0 100% /snap/core18/1944
/dev/loop4                                32M   32M     0 100% /snap/snapd/10492
/dev/loop3                                65M   65M     0 100% /snap/gtk-common-themes/1514
/dev/loop5                               163M  163M     0 100% /snap/gnome-3-28-1804/145
tmpfs                                    1.2G  4.0K  1.2G   1% /run/user/1000
tmpfs                                    1.2G   24K  1.2G   1% /run/user/124
```

**Comment by svpernova09** [comment]
Asking questions is 100% ok! Might be better to ask these types in #homestead on Discord though. 

TL;DR

PostgreSQL in Homestead saves the DB files in the same place it normally does. MySQL has this LVM volume so it can easily be backed up via the backup feature, also so users can push as much data into a DB as their Host has storage access for. (Although I believe we cap Homestead's primary filesystem to 128GB, that could be outdated and providers might have their own cleverness around getting past that.
